src/Stryker.Core/Stryker.Core/Baseline/Providers/AzureFileShareBaselineProvider.cs

```diff
@@ -53,15 +56,16 @@ public async Task<JsonReport> Load(string version)
 
         public async Task Save(JsonReport report, string version)
         {
-            var fileUrl = $"{_options.AzureFileStorageUrl}/{_outputPath}/{version}/stryker-report.json";
-            var url = new Uri($"{fileUrl}?comp=range&sv={_options.AzureFileStorageSas}");
```

Comment:
Since we would no longer inject sv= in front of the key this makes the change technically backwards incompatible, we will have to discuss if we want to make a breaking change at this point. I will get back to you on that. We don't have any analytics on if users would typically include the ?sv= in the key or not. If they don't (as our example does not), we would break everyone's setup.

Reply:
Yeah that’s a good point. 

From my point of view, you cannot use the later versions anyway due to the SAS has changed. 
you cannot use the old token generation either as in a previous PR we have removed the required headers. 

I’ll update the docs as part of this PR